### ISSUE TITLE
Customizable class names

### ISSUE THREAD

**Issue by mrksbnch** [frappe/charts#54]
What about an option that allows users to set custom class names and change the default ones, mostly for integrating with an existing design system?

**Comment by pratu16x7** [comment]
Not sure how the latter ("change the default ones") would work.

**Comment by mrksbnch** [comment]
Currently, the class names for the chart are hard coded, e.g. here you are settings the class names "x" and "axis" (https://github.com/frappe/charts/blob/master/src/scripts/charts/AxisChart.js#L89). As those class names allow you to customize the styles of a chart and some people prefer to use specific naming conventions for all of their class names it'd be helpful to change these. 

This will also minimize the risk of class name collision, e.g. if a project already uses "x" as a class name.

**Comment by pratu16x7** [comment]
Wouldn't that also imply a change in the stylesheet?

**Comment by mrksbnch** [comment]
Yes, the people who like to change the class names, e.g. due to a conflict with existing classes or because they want to use an existing naming convention / framework to style specific parts of the chart, would also need to adapt the stylesheet to their needs (which is probably the reason why they want/need to change the class names in the first place). 

For all others that don't use that "feature" nothing changes and they can keep using the default stylesheet of frappe. But I assume that most people will need to adjust the stylesheet anyway to e.g. match an existing design style guide.
